installers/kubectl/client-setup.sh

```diff
@@ -14,6 +14,40 @@
 # This script should be run after the operator has been deployed
 export PGO_OPERATOR_NAMESPACE="${PGO_OPERATOR_NAMESPACE:-pgo}"
 PGO_USER_ADMIN="${PGO_USER_ADMIN:-pgouser-admin}"
+PGO_CLIENT_VERSION="${PGO_CLIENT_VERSION:-v4.3.0}"
+PGO_CLIENT_URL="https://github.com/CrunchyData/postgres-operator/releases/download/${PGO_CLIENT_VERSION}"
+LINUX_BIN="pgo"
+DARWIN_BIN="pgo-mac"
+VALID_OS_OPTIONS="Linux, Darwin"
+
+# Checks operating system and determines which binary to download
+UNAME_RESULT=$(uname)
+if [ "${UNAME_RESULT}" == "Linux" ]
+then
+    BIN_NAME="${LINUX_BIN}"
+elif [ "${UNAME_RESULT}" == "Darwin" ]
+then
+    BIN_NAME="${DARWIN_BIN}"
+else
+    echo "${UNAME_RESULT} is not supported, valid operating systems are: ${VALID_OS_OPTIONS}"
+    echo "Exiting..."
+    exit 1
+fi
+
+# Creates the output directory for files
+OUTPUT_DIR="${HOME}/.pgo/${PGO_OPERATOR_NAMESPACE}"
+mkdir -p "${OUTPUT_DIR}"
+# lock down the directory to just this user
+chmod a-rwx,u+rwx "${OUTPUT_DIR}"
+
+echo "Operating System found is ${UNAME_RESULT}. Downloading ${BIN_NAME} client binary..."
+
+FULL_PATH="${PGO_CLIENT_URL}/${BIN_NAME}"
+curl -L "${FULL_PATH}" -o "${OUTPUT_DIR}/${BIN_NAME}"
+chmod 0755 "${OUTPUT_DIR}/${BIN_NAME}"
+
+echo "Binary downloaded to ${OUTPUT_DIR}/${BIN_NAME}, please update your bashrc with the following:"
+echo "export PATH=${OUTPUT_DIR}/${BIN_NAME}:\$PATH"
```

Comment:
I'd put this at the end with all the other stuff. Easy to copy and paste.

don't need the echo then to say it's downloaded.